providers/globus_test.go

```diff
@@ -0,0 +1,101 @@
+package providers
```

Comment:
Take a peek at `keycloak_test.go` or `gitlab_test.go` -- we are moving to Ginkgo tests for any new tests in the project.